.ruby-version

```diff
@@ -1 +1 @@
-2.6.6
+2.7.3
```

Comment:
We bumped `agra` to `2.7.4` due to a security fix, should we use that version instead?